docs/enterprise/getstarted/migrate-from-oss.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Migrate from lakeFS OSS
+description: How to migrate from lakeFS OSS to lakeFS Enterprise
+parent: Get Started
+grand_parent: lakeFS Enterprise
+nav_order: 203
+---
+
+# Migrate From lakeFS Open-Source to lakeFS Enterprise
```

Comment:
@Isan-Rivkin, I'll be happy to collaborate on this section. 
Can you please suggest your edits and correct the flow or terminology if needed?